### ISSUE TITLE
Can't remove folder which has error

### ISSUE THREAD

**Issue by 9helix** [syncthing/syncthing#9176]
I'm running version 1.25.0 on Windows 10. I by mistake included quotes in my folder path name when adding a folder through web GUI. When I ran the folder I got these errors:
```
2023-10-15 21:12:30: Loading ignores: CreateFile \\?\C:\Windows\system32\"D:\Documents\Obsidian Vault"\.stignore: The filename, directory name, or volume label syntax is incorrect.

2023-10-15 21:12:30: Failed to create folder root directory CreateFile \\?\C:\Windows\system32\"D:\Documents\Obsidian Vault": The filename, directory name, or volume label syntax is incorrect.

2023-10-15 21:12:30: Error on folder cjckj-e27rf: CreateFile \\?\C:\Windows\system32\"D:\Documents\Obsidian Vault": The filename, directory name, or volume label syntax is incorrect.
```
Then I added a new folder with a correct path and it worked fine. Then I wanted to delete the erroneous folder from before by editing it in GUI. But the moment I pressed Edit it poped-up an error
```
Connection Error
Syncthing seems to be experiencing a problem processing your request. Please refresh the page or restart Syncthing if the problem persists.
```
and I couldn't click the Remove button. Now I can't remove it even if I refresh the site as the Error pops up again and it won't let me click the Remove button.

**Comment by er-pa** [comment]
When opening the folder editor of a faulty folder (at least with a faulty path), `GET "/rest/db/ignores?folder=hcufa-fecvk"` tosses a 500 status, which probably triggers the dialog.

You can always edit the config.xml file yourself to remove the faulty folder, that ofc till this is otherwise fixed:)

**Comment by clivechentutu** [comment]
Same bug

**Comment by 9helix** [comment]
It is located in `C:\Users\username\AppData\Local\Syncthing\config.xml`. Locate the folder's label inside the file and delete its associating `folder` tag. After restarting Snytching the problematic folder should disappear.

**Comment by Liamcsail** [comment]
> It is located in `C:\Users\username\AppData\Local\Syncthing\config.xml`. Locate the folder's label inside the file and delete its associating `folder` tag. After restarting Snytching the problematic folder should disappear.它位于 `C:\Users\username\AppData\Local\Syncthing\config.xml` 。在文件内找到文件夹的标签并删除其关联的 `folder` 标记。重新启动 Snytching 后，有问题的文件夹应该会消失。

thx so much！！problem done！！

**Comment by calmh** [comment]
This was fixed in #9254

**Comment by calmh** [comment]
Same as #9253
